fix determining path to top-level directory,
since run_comebin.sh is located in bin/run_comebin.sh
--- bin/run_comebin.sh.orig	2024-06-04 16:27:24.373968000 +0200
+++ bin/run_comebin.sh	2024-06-04 16:27:38.413830000 +0200
@@ -25,6 +25,7 @@
 	echo "";}
 
 run_file_path=$(dirname $(which run_comebin.sh))
+run_file_path=$(dirname $(dirname $(which run_comebin.sh)))
 
 if [[ $? -ne 0 ]]; then
 	echo "cannot find run_comebin.sh file - something went wrong with the installation!"
